src/ExponentialMap.jl

```diff
@@ -484,6 +501,35 @@ function σ(d::AbstractVector{N},
     return eprojmap.projspmexp.L * daux
 end
 
+"""
+    isbounded(eprojmap::ExponentialProjectionMap)::Bool
+
+Determine whether an exponential projection map is bounded.
+
+### Input
+
+- `eprojmap` -- exponential projection map
+
+### Output
+
+`true` iff the exponential projection map is bounded.
+
+### Algorithm
+
+We first check if the wrapped set is bounded or if the left or right projection
+matrix is zero.
+Otherwise, we check boundedness via [`isbounded_unit_dimensions`](@ref).
+"""
+function isbounded(eprojmap::ExponentialProjectionMap)::Bool
+    if isbounded(eprojmap.X)
+        return true
+    end
+    if iszero(eprojmap.projspmexp.L) || iszero(eprojmap.projspmexp.R)
```

Comment:
Swap these conditions? These `if` clauses, or write it like this?

```julia
if iszero(eprojmap.projspmexp.L) || iszero(eprojmap.projspmexp.R) || isbounded(eprojmap.X)
```m
Even if it can fail most times, in general it will be easier to check if `L` or `R` are empty than boundedness of `X`.

Reply:
Depends: if `X` is of basic set type different from `HPolyhedron`, the check is constant-time. But okay!